src/utils/thermodynamics_overrides.jl

```diff
@@ -0,0 +1,55 @@
+"""
```

Comment:
Rename to 'refstate_thermodynamics` (to make clear this is just about the reference state)